docs/guide/security_checklist.md

```diff
@@ -0,0 +1,118 @@
+# Security checklist
+
+When getting ready to go live with your project for the first time, or when re-launching it,
+you need to make sure your setup is secure.
+
+!!! caution
+
+    Security is an ongoing process. After going live, you should pay attention to security advisories
+    released via [your service portal,](https://support.ibexa.co/)
+    or via [Security advisories](https://developers.ibexa.co/security-advisories) if you're not a subscriber.
+
+## Symfony
+
+### `APP_SECRET`
+
+`APP_SECRET` needs to be a strong, random, securely stored value.
+
+- Do not use a default value like `ff6dc61a329dc96652bb092ec58981f7` or `ThisTokenIsNotSoSecretChangeIt`
+- The secret must be secured against unwanted access. Do not commit it to version control.
+- The secret must be long enough. 32 characters is minimum, longer is better.
+
+!!! tip
+    
+    This will generate a 64-character-long secure random secret:
```

Comment:
```suggestion
    The following command will generate a 64-character-long secure random value:
```
@dabrt I'd like to keep "secure" here since not all random generators are secure (unpredictable).